_schemaV2.graphql

```diff
@@ -9200,6 +9201,7 @@ type Query {
   # Find PartnerStats
   analyticsPartnerStats(partnerId: String!): AnalyticsPartnerStats
   marketingCollections(
+    category: String
```

Comment:
This seems to come from kaws but not sure why it wasn't picked up from previous PR.